front/components/items/Appointment/models.ts

```diff
@@ -0,0 +1,18 @@
+import { strapiAppointmentQuery } from 'lib/strapi/appointments/queryType';
+
+export type filters = {
+    [Keys: string]: Object;
+};
+export type propsOfFormingDate = {
+    chosenDate: Date;
+};
+export type propsAppointment = {
```

Comment:
keep one convention. Above and below you have propsOfXYZ, there you have propsXYZ